client/my-sites/plans/navigation.jsx

```diff
@@ -163,4 +169,14 @@ class PlansNavigation extends React.Component {
 	}
 }
 
-export default localize( PlansNavigation );
+export default connect( state => {
+	const siteId = getSelectedSiteId( state );
+	const site = getSite( state );
```

Comment:
Oooops. :stuck_out_tongue: Need to pass the `siteId`. :) Other than that :shipit:.

Reply:
🤦‍♂️ Thanks for catching that. Fixed in 1011a7dd85269390cc3b400d6c816ab92d96b411